packages/catalog-search/src/styles/_SearchSuggestions.scss

```diff
@@ -13,53 +13,84 @@
   margin: 5px 10px 5px 10px;
   padding: 8px;
   color: #000;
+  font-size: 75%;
   white-space: pre-wrap;
-    &:hover{
-      color:white;
-      background-color:#000000 ;
-      text-decoration: none;
-  }
 
-    &:hover span {
-      color:white;
+  &:hover {
+    color: white;
+    background-color: #000000;
+    text-decoration: none;
   }
-  em {
-  font-weight: bold;
-  font-style: normal;
-  &:hover{
+
+  &:hover span {
     color: white;
   }
-  &:not(:first-child) {
-    margin-left: 4px; // add space between _highlightResult.title.value hits
+
+  em {
+    font-weight: bold;
+    font-style: normal;
+
+    &:hover {
+      color: white;
+    }
+
+    &:not(:first-child) {
+      margin-left: 4px; // add space between _highlightResult.title.value hits
+    }
   }
-}
-  div{
+
+  div {
     display: flex;
   }
-  .authoring-org-badge{
+
+  .authoring-org-badge {
     line-height: 1.5;
   }
 
-  .program-type{
+  .program-type {
     font-size: .9rem;
     margin-bottom: 0px;
   }
 
 }
 
+.prequery-item {
+  margin: 5px 10px 5px 10px;
+  color: #000;
```

Comment:
I would replace the hardcoded hex values with `$white` or `white`

Reply:
this value is actually closer to black. I will leave this as is since it is the convention used in the current file.